### PR TITLE
Fix few minor typos

### PR DESCRIPTION
**Pull request by tsachiherman**
## Summary

Fix several typos.

## Test Plan

No tests needed for fixing in-code comments.

### CODE REVIEW

**Review by algonautshant — DISMISSED** [review]
LGTM

File: ledger/acctupdates.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func (e *RoundOffsetError) Error() string {
 }
 
 // StaleDatabaseRoundError is generated when we detect that the database round is behind the accountUpdates in-memory dbRound. This
-// should never happen, since we update the database first, and only upon a successfull update we update the in-memory dbRound.
+// should never happen, since we update the database first, and only upon a successful update we update the in-memory dbRound.
```
Comment (algonautshant) [inline ledger/acctupdates.go:244]: Similar typos:
catchup/catchpointService.go:410:		// give a rank to the download, as the download was **successfull**.
network/wsPeer.go:439:				// do nothing. writing was **successfull**.
scripts/buildhost/start_ec2_instance.sh:13:# Upon **successfull** execution, the following files would be created:
test/framework/fixtures/libgoalFixture.go:105:// nodeExitWithError is a callback from the network indicating that the node exit with an error after a **successfull** startup.

**Review by algonautshant — DISMISSED** [review]

File: ledger/acctupdates.go (hunk 1 above)
Comment (tsachiherman) [reply ledger/acctupdates.go:244]: fixed.

**Review by algonautshant — APPROVED** [review]

File: data/basics/userBalance.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ const (
 	// NotParticipating indicates that the associated account neither participates in the consensus, nor recieves rewards.
 	// Accounts that are marked as NotParticipating cannot change their status, but can receive and send Algos to other accounts.
 	// Two special accounts that are defined as NotParticipating are the incentive pool (also know as rewards pool) and the fee sink.
-	// ( these two accounts also have additional Algo transfer restrictions )
+	// These two accounts also have additional Algo transfer restrictions
```
Comment (winder) [inline data/basics/userBalance.go:40]: nit, but as long as we're fixing typos...
```suggestion
	// These two accounts also have additional Algo transfer restrictions.
```

**Review by nicholasguoalgorand — APPROVED** [review]

File: data/basics/userBalance.go (hunk 2 above)
Comment (tsachiherman) [reply data/basics/userBalance.go:40]: as before, I'll fix this on next PR.

**Pull request merged**